config/vufind/combined.ini

```diff
@@ -33,6 +33,15 @@
 ;         the side column. If set to false, side recommendations will be suppressed
 ;         (default). Adding the same recommendation module to multiple sections is
 ;         unsupported.
+; include_recommendations_noresults = If set to true, standard 'top'
+;         recommendations will be displayed at the top of the column. If set to an
+;         array of recommendation settings (as per searches.ini), the specified
+;         recommendations will be loaded into the results column. If set to false,
+;         noresults recommendations will be suppressed (default). Adding the same
```

Comment:
Does the "Adding the same recommendation module to multiple sections is unsupported." note really only apply if using include_recommendations_noresults_side, or is the restriction broader than that? I'm wondering if that note should be moved down.

Reply:
You're right, that was another editing error.  Removed.